cli/pkg/app/exec.go

```diff
@@ -155,7 +155,9 @@ func listenOnStderr(c *cc.CommonCtx, wg *sync.WaitGroup, stderr io.ReadCloser) {
 
 	for {
 		line, err := scanner.ReadBytes('\n')
```

Comment:
Looks like the error from ReadBytes is being ignored. I'm surprised the linter doesn't warn about it.